data_hub_api/docmaps/provider.py

```diff
@@ -84,87 +88,90 @@ def get_docmap_assertions_value_for_preprint_under_review_step(
+        'outputs': [{
+            'identifier': manuscript_id,
+            'versionIdentifier': '',
             'type': 'preprint',
-            'doi': query_result_item['preprint_doi'],
-            'versionIdentifier': ''
-        },
-        'status': 'peer-reviewed'
+            'doi': elife_doi,
+            'url': elife_doi_url,
+            'content': [{
+                'type': 'web-page',
+                'url': f'{ELIFE_REVIEWED_PREPRINTS_URL}{manuscript_id}'
+            }]
+        }]
     }]
 
 
-def get_docmap_assertions_value_from_query_result(
-    step_number: int,
+def get_docmap_inputs_value_for_review_steps(
     query_result_item: dict
 ) -> list:
-    if step_number == 0:
-        return get_docmap_assertions_value_for_preprint_manuscript_published_step(
+    return [{
+        'type': 'preprint',
+        'doi': query_result_item['preprint_doi'],
+        'url': query_result_item['preprint_url'],
+    }]
+
+
+def get_docmaps_step_for_under_review_status(
+    query_result_item
+):
+    return {
+        'actions': get_docmap_actions_value_for_preprint_under_review_step(
             query_result_item=query_result_item
-        )
-    if step_number == 1:
-        return get_docmap_assertions_value_for_preprint_under_review_step(
+        ),
+        'assertions': get_docmap_assertions_value_for_preprint_under_review_step(
             query_result_item=query_result_item
-        )
-    if step_number == 2:
-        return get_docmap_assertions_value_for_preprint_peer_reviewed_step(
+        ),
+        'inputs': get_docmap_inputs_value_for_review_steps(
             query_result_item=query_result_item
         )
-    return []
+    }
 
 
-def get_single_actions_value_for_preprint_manuscript_published_step(
-    preprint_doi: str,
-    preprint_published: str
-) -> dict:
-    return {
-        'participants': [],
-        'outputs': [{
+def get_docmap_assertions_value_for_preprint_peer_reviewed_step(
+    query_result_item: dict
+) -> list:
+    return [{
+        'item': {
             'type': 'preprint',
-            'doi': preprint_doi,
-            'url': f'{DOI_ROOT_URL}{preprint_doi}',
-            'published': preprint_published,
+            'doi': query_result_item['preprint_doi'],
             'versionIdentifier': ''
-        }]
-    }
+        },
+        'status': 'peer-reviewed'
+    }]
 
 
-def get_single_actions_value_for_preprint_under_review_step(
-    manuscript_id: str,
-    elife_doi: str,
-    elife_doi_url: str
-) -> dict:
-    return {
-        'participants': [],
-        'outputs': [{
-            'identifier': manuscript_id,
-            'versionIdentifier': '',
-            'type': 'preprint',
-            'doi': elife_doi,
-            'url': elife_doi_url,
-            'content': [{
-                'type': 'web-page',
-                'url': f'{ELIFE_REVIEWED_PREPRINTS_URL}{manuscript_id}'
-            }]
-        }]
-    }
+def get_outputs_type_for_peer_reviewed_step(
```

Comment:
I think it would be good to have a separate test for this function to test with a few tag combinations.
The default should probably be `None`.
Only if we found a related tag, we should recognise it as a `review-article`.

Reply:
If there is unrelated tags probably we need to filter out these data for peer-reviewed step. Because according to examples it's only expecting two types (`review-article` and `evaluation-summary`) and we don't need  `None` types. What do you think?
And what should be the related tag for `review-article`?

Reply:
Yes, `None` would signify that we don't want to include them alternatively we could raise an error. But not including them seems okay in this case.
(And there is the question whether we want to include author responses in the reviewed-preprint step or not)